### ISSUE TITLE
Python3 compatibility problem

### ISSUE THREAD

**Issue by mozbugbox** [rk700/PyMuPDF#47]
The Page.getPixmap() in the tutorial and API doc does not exist in current master.

Besides, the `object.__getattr__` in `fitz.py` does not exist in Python3.

**Comment by mozbugbox** [comment]
actually, it's the problem with python3 compatible thing. I cannot when calling page.getPixmap or document.getPagePixmap, I got:
```  File "/usr/lib/python3/dist-packages/fitz/fitz.py", line 101, in <lambda>
    __getattr__ = lambda self, name: _swig_getattr(self, Document, name)
  File "/usr/lib/python3/dist-packages/fitz/fitz.py", line 74, in _swig_getattr
    return _swig_getattr_nondynamic(self, class_type, name, 0)
  File "/usr/lib/python3/dist-packages/fitz/fitz.py", line 69, in _swig_getattr_nondynamic
    return object.__getattr__(self, name)
AttributeError: type object 'object' has no attribute '__getattr__'
```
when change `__getattr__` to `__getattribute__`, I got 
```
  File "/usr/lib/python3/dist-packages/fitz/fitz.py", line 101, in <lambda>
    __getattr__ = lambda self, name: _swig_getattr(self, Document, name)
  File "/usr/lib/python3/dist-packages/fitz/fitz.py", line 74, in _swig_getattr
    return _swig_getattr_nondynamic(self, class_type, name, 0)
  File "/usr/lib/python3/dist-packages/fitz/fitz.py", line 69, in _swig_getattr_nondynamic
    return object.__getattribute__(self, name)
AttributeError: 'Document' object has no attribute 'getPagePixmap'
```

**Comment by JorjMcKie** [comment]
Looks awkward.
Methods getPixmap() and getPagePixmap() are contained in setup file ``utils.py`` and get connected with ``fitz.py`` during invocation of ``__init__.py``, i.e. when you do ``import fitz``.
Right after the import, you should see those methods when you do e.g. a ``dir(fitz.Document)`` like so:
```
Python 3.5.1 (v3.5.1:37a07cee5969, Dec  6 2015, 01:54:25) [MSC v.1900 64 bit (AMD64)] on win32
Type "help", "copyright", "credits" or "license" for more information.
>>> import fitz
>>> dir(fitz.Document)
['__class__', '__del__', '__delattr__', '__dict__', '__dir__', '__doc__', '__eq__', '__format__', '__ge__', '__getattr__', '__getattribute__', '__gt__', '__hash__', '__init__', '__le__', '__lt__', '__module__', '__ne__', '__new__', '__reduce__', '__reduce_ex__', '__repr__', '__setattr__', '__sizeof__', '__str__', '__subclasshook__', '__swig_destroy__', '__swig_getmethods__', '__swig_setmethods__', '__weakref__', '_dropOutline', '_getMetadata', '_getPageCount', '_loadOutline', '_needsPass', '_readPageText', '_select', 'authenticate', 'close', 'getPagePixmap', 'getPageText', 'getPermits', 'getToC', 'initData', 'loadPage', 'needsPass', 'outline', 'pageCount', 'save', 'select']
>>>
```
Similarly for a ``dir(fitz.Page)``. If that's not the case (whether Python3 or Python2) something went wrong with your installation. Maybe it is a problem with the ``setup.py`` file. Its very last 2 lines must read:

```
      ext_modules = [module],
      py_modules = ['fitz.fitz', 'fitz.utils'])
```

**Comment by JorjMcKie** [comment]
BTW: why do you say "Python3 compatibility issue"?
Do you have a correctly working version under Python2? And only Python3 fails?
From your error messages I cannot see anything that hints at an error **specific** for Python3 ...
Please do keep us current about your findings.

**Comment by mozbugbox** [comment]
You were right that the utils.py was not properly installed. After repack and re-install the python3-fitz package, the problem was fixed magically.

I was using `stdeb` to debianize the python package.

I did have both python2 and python3 version of fitz installed. While I was test the problem, the python2 version of python-fitz worked fine.

I'm not sure what was happening but it's fixed now. Thank you very much!
